UPGRADING.rst

```diff
@@ -15,7 +15,12 @@ have too many entries in this file.
 
 This PR introduces two new deployment-specific environment variables,
 ``AZUL_TDR_SERVICE_URL`` and ``AZUL_SAM_SERVICE_URL``. Copy the settings for
-these variables from the sandbox deployment to your personal deployment.
+these variables from the example deployment to your personal deployment.
+
+Service accounts must be registered and authorized with SAM for integration
+tests to pass. See the last two paragraphs of section 3.2 of the README for
```

Comment:
Make subsection in README and hyperlink to it if possible.

Reply:
Don't think there's any way to link to .md from .rst